servicemesh-release-notes/topics/rn-known-issues.adoc

```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
 
+Graph layout - The layout for the Kiali graph may render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions).  Because it is difficult if not impossible to create a single layout that could render nicely for every situation, Kiali offers a choice of several different layouts.  To choose a different layout you can choose a different Layout Schema from the Graph Settings menu.
+
+https://issues.jboss.org/browse/MAISTRA-13[MAISTRA-13] - Installer should determine release version from installed components.  At present the installer queries the yaml file, however if the yaml has been modified the installer is not able to remove an older version.
 
-https://issues.jboss.org/browse/MAISTRA-21[MAISTRA-21] - Increase elasticsearch memory requirement.
-The current default in the installer is 512Mi however this appears to be too low for tracing.
+*Workaround* - If the installation fails because of a previous, older installation, then use the following workaround:
 
-*Workaround* - Include the following in the custom resource definition file.
 ```
-spec:
-  jaeger:
-    elasticsearch_memory: 1Gi
+oc delete namespace devex istio-operator istio-system bookinfo logging foo bar
+
+oc delete csr istio-sidecar-injector.istio-system
+
+oc get crd  | grep istio | awk '{print $1}' | xargs oc delete crd
+
+oc get mutatingwebhookconfigurations  | grep istio | awk '{print $1}' | xargs oc delete mutatingwebhookconfigurations
+
+oc get validatingwebhookconfiguration  | grep istio | awk '{print $1}' | xargs oc delete validatingwebhookconfiguration
+
+oc get clusterroles  | grep istio | awk '{print $1}' | xargs oc delete clusterroles
+
+oc get clusterroles  | grep kiali | awk '{print $1}' | xargs oc delete clusterroles
+
+oc get clusterrolebindings  | grep istio | awk '{print $1}' | xargs oc delete clusterrolebindings
+
+oc get clusterrolebindings  | grep kiali | awk '{print $1}' | xargs oc delete clusterrolebindings
 ```
+
+https://github.com/kiali/kiali/issues/507[Kiali-507] Kiali does not support Internet Explorer 11.  This is because the underlying frameworks do not support Internet Explorer.  In order to access the Kiali console you should be using the two most recent versions of the Chrome, Edge, Firefox or Safari browser.
+
+https://issues.jboss.org/browse/KIALI-1284[KIALI-1284] In Istio, a Workload can be any pod or group of pods, regardless where they originate from. They may come from Kubernetes Deployments, Replica Sets or even as a single "orphan" pod. In Kiali the current assumption is that a Workload comes from a Deployment. This should represent the vast majority of the cases.
+
+https://issues.jboss.org/browse/KIALI-1570[KIALI-1570]
+When a graph is loading in the Kiali console, it will initially say that the graph is empty instead of displaying a message that the graph is loading.
```

Comment:
Maybe "...a message that the graph is empty is displayed instead of a message that the graph is loading."